Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+# v1.11.3
+- [changed] Changed the default log level to `notice` instead of `warning` (#5196).
```

Comment:
This only applies to non-Apple-platform users so it doesn't belong in this changelog.

Reply:
Removed.